package.json

```
@@ -26,17 +26,17 @@
     "build": "rimraf dist/* && rollup -c",
     "watch": "rollup -c rollup.config.self.js -w ",
     "build-self": "rimraf dist/* && rollup -c rollup.config.self.js",
-    "lint": "tslint -c ./tslint.json src/*.ts __tests__/*.ts ./*.js",
+    "lint": "npx eslint .",
```

```suggestion
    "lint": "eslint .",
```

`npx` is not needed; `bin` is accessible within `scripts`